sources/Outputs/src/DydVRRemote.cpp

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) 2023, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+#include "DydVRRemote.h"
+
+#include "DydCommon.h"
+#include "OutputsConstants.h"
+
+namespace dfl {
+namespace outputs {
+
+void
+DydVRRemote::writeVRRemotes(boost::shared_ptr<dynamicdata::DynamicModelsCollection>& dynamicModelsToConnect,
+                            const algo::GeneratorDefinitionAlgorithm::BusGenMap& busesRegulatedBySeveralGenerators,
+                            const algo::HVDCLineDefinitions::BusVSCMap& vscBusVSCDefinitionsMap,
+                            const std::string& basename) {
+    using BusId = std::string;
+    std::unordered_set<BusId> regulatedBusIds;
+    for (const auto& busToGenerator : busesRegulatedBySeveralGenerators)
+        regulatedBusIds.insert(busToGenerator.first);
+    for (const auto& busToVSC : vscBusVSCDefinitionsMap)
+        regulatedBusIds.insert(busToVSC.first);
+
+    for (const BusId& regulatedBusId : regulatedBusIds) {
+        std::string id = constants::modelSignalNQprefix_ + regulatedBusId;
+        boost::shared_ptr<dynamicdata::BlackBoxModel> blackBoxModelVRRemote =
+                                                        helper::buildBlackBox(id, "VRRemote", basename + ".par", id);
+        dynamicModelsToConnect->addModel(blackBoxModelVRRemote);
+        dynamicModelsToConnect->addConnect(id, "vrremote_URegulated", constants::networkModelName, regulatedBusId + "_U_value");
```

Comment:
the connections to the generators/hvdc are missing
![image](https://github.com/dynawo/dynaflow-launcher/assets/47146813/6c0cac90-a464-4583-849b-fb4bd6fd86a0)

Reply:
Yes, this is the point of this development